src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+/**
+ * 
+ * @author joellim
```

Comment:
you can possibly remove the `@author` tag; your git commits identify you any way :-)